src/lib/transaction_snark/test/account_timing/account_timing.ml

```diff
@@ -1563,4 +1563,84 @@ let%test_module "account timing check" =
                   ~state_view ledger parties
               in
               check_zkapp_failure Transaction_status.Failure.Overflow result ) )
+
+    let%test_unit "zkApp command, create timed account with wrong authorization"
```

Comment:
Could you write a test for correct authorization and a test for the new case (fails when the account is already timed)?